src/HornedBeast.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+class HornedBeast extends React.Component {
+  render() {
+    return(
+      <article>
+        <h2>{this.props.name}</h2>
+        <img 
+        src={this.props.imageUrl} 
+        alt={this.props.name}
+        title={this.props.name}/>
```

Comment:
Really good work! I am liking' that image too! :)

- for lines 9-11 we can indent 👍